sdk/Notifo.SDK.Core/NotifoMobilePush/NotifoMobilePushImplementation.ios.cs

```diff
@@ -82,7 +82,7 @@ private async Task PullPendingNotificationsAndTrack()
                     continue;
                 }
 
-                await ShowLocalNotificationAsync(notification);
+                ShowLocalNotificationAsync(notification).Forget();
```

Comment:
Why do we not await here? Comment?

Reply:
If there is a reason, we should comment it.

Reply:
there's just no need to wo wait. 
We're only enriching the notifications and push them to the ios scheduler.

Reply:
Yes, but in general it is bad practice. For example, if you would have 100 pending items, you would make the other requests (e.g. downloading images) in parallel and potentially exhaust available resources. In server code this can bring your system down. Therefore it catches attention.

Reply:
I agree, I was indeed not thinking of the images that have to be downloaded. 
With the very restrictive memory limit of the notification extension this could really a bigger issue.
I'm just also thinking of the 30 sec limit for the extension. With 100 messages this could also be an issue. But I guess there is a point where the extension just doesn't make sense anymore.

Reply:
Just add a comment, that would help. There was not a single comment and it was difficult to understand all the assumptions when we had a look to the code the first time.